src/config/SettingsConfig.hh

```diff
@@ -34,8 +35,12 @@ public:
 	void setValueForSetting(std::string_view setting, std::string_view value);
 	void removeValueForSetting(std::string_view setting);
 
+	void setShortcuts(Shortcuts& shortcuts_) { shortcuts = &shortcuts_; };
```

Comment:
I don't love this solution and preferred to use references, but they need to be instantiated in the constructor. Object creation as it is is messy and confusing.